### PR TITLE
Fully hardcoded main page

### PR DESCRIPTION
**Pull request by smruthibalajee**
**Description:** Created the frontend components of the main page for a logged out user. All cluster values are hardcoded.

**Key Features:**

- Card resizing: When the user expands or reduces the page, the card's image zoom will adjust accordingly

- Card carousel: Cards can be moved to the left or right one at a time when the user clicks on the arrow controls. Created Javascript function and utilized the carousel functionality in Bootstrap to do this.

**Demo:**
ripple-main-page

**Nice to haves (if time permits):**

- Progress bar over each card cluster indicating how much a user has scrolled to the left or right.

- View all button indicates the total number of businesses in the category (taken from database).

- Fix the left and right carousel controls to be more clear and similar to the prototype version. Hide the corresponding button if the user has gone completely to the left or completely to the right.

### CODE REVIEW

File: ripple/src/main/webapp/style.css [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+.card {
+    width: 300px;
+    height: 200px
+}
+
+#cardCarousel {
+    width: 1090px;
+}
+
+/**make carousel responsive to screen size **/
```
Comment (peter14f) [inline ripple/src/main/webapp/style.css]: nit: https://google.github.io/styleguide/jsguide.html#formatting-block-comment-style

File: ripple/target/portfolio-1/script.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+
+/**Carousel function that allows cards to move to the left or right by one.*/
```
Comment (peter14f) [inline ripple/target/portfolio-1/script.js:2]: ditto

File: ripple/target/portfolio-1/style.css [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+.card {
+    width: 500px;
+    height: 350px
+}
+
+#cardCarousel {
+    width: 1090px;
+}
+
+/**make carousel responsive to screen size **/
```
Comment (peter14f) [inline ripple/target/portfolio-1/style.css:16]: ditto

File: ripple/src/main/webapp/style.css (hunk 1 above)
Comment (smruthibalajee) [reply ripple/src/main/webapp/style.css]: fixed!

File: ripple/target/portfolio-1/style.css (hunk 3 above)
Comment (smruthibalajee) [reply ripple/target/portfolio-1/style.css:16]: fixed!

File: ripple/target/portfolio-1/script.js (hunk 2 above)
Comment (smruthibalajee) [reply ripple/target/portfolio-1/script.js:2]: fixed!

**Pull request merged**